protos/flyteidl/service/admin.proto

```diff
@@ -155,13 +151,19 @@ service AdminService {
         get: "/api/v1/launch_plans/{id.project}/{id.domain}"
       }
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "Fetch existing launch plan definitions matching input filters."
+    };
   }
 
   rpc UpdateLaunchPlan (flyteidl.admin.LaunchPlanUpdateRequest) returns (flyteidl.admin.LaunchPlanUpdateResponse) {
     option (google.api.http) = {
       put: "/api/v1/launch_plans/{id.project}/{id.domain}/{id.name}/{id.version}"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "Update the status of an existing launch plan definition."
```

Comment:
Since this is _really_ the core of the entire platform, I think this is a natural space to explain the semantics of the platform.  Most endpoints are pretty self-explanatory so they don't need much text, but for things like update launch plan, it would be good to describe what setting a launch plan to active means.

Also, describing the idempotent behavior of our create endpoints would be helpful.

Reply:
sure, done for both